app/src/main/res/layout/fragment_login.xml

```diff
@@ -149,4 +151,4 @@
         </LinearLayout>
 
     </LinearLayout>
```

Comment:
You replaced a ScrollView, so this should be wrapped in ScrollView or NestedScrollView to be specific